README.md

```diff
@@ -77,37 +71,11 @@ to log in.
 
 Once you have logged in, you can make your user an admin, by running
 
-    python manage.py add_admin <your user>
+    docker-compose -f docker-compose.dev.yml run webserver python2 manage.py add_admin <your user>
```

Comment:
in these run commands we should add the `--rm` flag to remove the container after it finishes

Reply:
ideally we should be using `-p` to set the project to `acousticbrainz` instead of the default `docker`. Perhaps adding in develop.sh would help with this?

Should we consider moving the main content of test.sh and develop.sh to brainzutils and have stubs in the projects? I'm not sure how to "install" it for development purposes, as brainzutils installs into the docker container